File: iniciante/1037/1037.cpp
// 1037 - Intervalo - beecrowd
// Iniciante - Nível 3

#include <iostream>
#include <string>

int main()
{
    double valor;
    std::cin >> valor;

    if(valor < 0 || valor > 100)
    {
        std::cout << "Fora de intervalo\n";
    }
    else
    {
        std::string intervalo;

        if     (valor <= 25)               intervalo = "[0,25]";
        else if(valor > 25 && valor <= 50) intervalo = "(25,50]";
        else if(valor > 50 && valor <= 75) intervalo = "(50,75]";
        else                               intervalo = "(75,100]";

        std::cout << "Intervalo " << intervalo << '\n';
    }

    return 0;
}

File: iniciante/1037/1037.py
# 1037 - Intervalo - beecrowd
# Iniciante - Nível 3

valor = float(input())

if valor < 0 or valor > 100:
    print("Fora de intervalo")
else:
    if valor <= 25:
        intervalo = "[0,25]"
    elif valor > 25 and valor <= 50:
        intervalo = "(25,50]"
    elif valor > 50 and valor <= 75:
        intervalo = "(50,75]"
    else:
        intervalo = "(75,100]"

    print(f"Intervalo {intervalo}")
